proposals/proposal_process.md

```diff
@@ -184,3 +184,58 @@ sponsored by a browser vendor) and a reviewing browser vendor are unable to
 reach consensus (with regard to either the API's concept or an individual
 aspect of it).  In this scenario, the reviewing vendor must choose a stance
 (which will very likely be "Opposed") to allow the API to move forward.
+
+## Review Process
+
+Reviewing new proposals is one of the most important and impactful roles of the
+Web Extensions Community Group. This ensures browsers have the opportunity to
+align on shared functionality when desirable for new functionality and helps
+prevent drift between the API surfaces. As such, it is critical to ensure
+timely reviews.
+
+### Authors
+
+Authors should request a review from at least one representative from each
+browser engine.
+
+When responding to feedback from reviewers, authors should resolve conversations
+if they feel they have been fully addressed. This makes it easy to see if there
+are outstanding comments. Authors can (and should) leave comments open if they
+would like further input from reviewers (e.g., "Is this what you were looking
+for").
+
+If reviewers haven't responded within a few business days, authors should ping
+the relevant reviewers.
+
+### Reviewers
+
+Reviewers should strive to respond to a review within one or two business days;
+this is not a strict SLO, but is a target -- we understand that this may not
+always be possible (e.g. if multiple reviewers are at a summit).
```

Comment:
Reworded to read a little more clearly and to use more common punctuation.

```suggestion
Reviewers should strive to respond to a review within one or two business days.
This is a target, not a strict SLO: we understand that this may not always be
possible (e.g. if multiple reviewers are at a summit).
```